.github/pull_request_template.md

```diff
@@ -1,33 +1,37 @@
 # Description
 
-*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this PR is changing it, and what motivated the change. If this is a breaking change, specify explicitly which APIs have been changed*
+*Replace this paragraph with a description of what this PR is doing. 
+If you're modifying existing behavior, describe the existing behavior, how this PR is changing it,
+and what motivated the change. If this is a breaking change, specify explicitly which APIs have been
+changed.*
 
 ## Checklist
 
 Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.
 
 - [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
-- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors and are passing (See [Contributor Guide]).
-- [ ] My PR does not decrease the code coverage, or I have __a very special case__ and explained on the PR description why this PR decreases the coverage.
-- [ ] I updated/added relevant documentation (doc comments with `///`) and updated/added examples in `doc/examples`.
-- [ ] I have formatted my code with `flutter format` and the `flutter analyze` does not report any problems.
-- [ ] I read and followed the [Flame Style Guide].
-- [ ] I have added a description of the change under `[next]` in `CHANGELOG.md`.
-- [ ] I removed the `Draft` status, by clicking on the `Ready for review` button in this PR.
-- [ ] I am willing to follow-up on review comments in a timely manner.
+- [ ] My PR includes tests for ALL new/updated/fixed functionality.
+- [ ] My PR does not decrease the code coverage.
+- [ ] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `doc:` etc).
+- [ ] I updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
+- [ ] I updated/added relevant examples in `examples`.
+- [ ] I have formatted my code with `flutter format` and `flutter analyze` does not report any problems.
```

Comment:
should we just replace this build is green to account for format, analyze, docs, tests, etc